.eslintrc

```diff
@@ -0,0 +1,43 @@
+{
```

Comment:
Some simple eslint rules - no need to be strict yet, but should try follow some guide.